--- SW/HFT-Core-Module_TrueSTUDIO/Drivers/STM32L4xx_HAL_Driver/Src/stm32l4xx_hal_rcc.c	2018-10-14 12:53:07.000000000 +0200
+++ SW/HFT-Core-Module_TrueSTUDIO/Drivers/STM32L4xx_HAL_Driver/Src/stm32l4xx_hal_rcc.c	2018-10-14 13:00:04.000000000 +0200
@@ -804,17 +804,8 @@
         assert_param(IS_RCC_PLLQ_VALUE(RCC_OscInitStruct->PLL.PLLQ));
         assert_param(IS_RCC_PLLR_VALUE(RCC_OscInitStruct->PLL.PLLR));
 
-        uint8_t pllsai1on = 0U, pllsai2on = 0U;
         /* Disable the main PLL. */
         __HAL_RCC_PLL_DISABLE();
-        if (READ_BIT(RCC->CR, RCC_CR_PLLSAI1ON)) {
-          CLEAR_BIT(RCC->CR, RCC_CR_PLLSAI1ON);
-          pllsai1on = 1U;
-        }
-        if (READ_BIT(RCC->CR, RCC_CR_PLLSAI2ON)) {
-          CLEAR_BIT(RCC->CR, RCC_CR_PLLSAI2ON);
-          pllsai2on = 1U;
-        }
 
         /* Get Start Tick*/
         tickstart = HAL_GetTick();
@@ -838,12 +829,6 @@
 
         /* Enable the main PLL. */
         __HAL_RCC_PLL_ENABLE();
-        if (pllsai1on) {
-          SET_BIT(RCC->CR, RCC_CR_PLLSAI1ON);
-        }
-        if (pllsai2on) {
-          SET_BIT(RCC->CR, RCC_CR_PLLSAI1ON);
-        }
 
         /* Enable PLL System Clock output. */
          __HAL_RCC_PLLCLKOUT_ENABLE(RCC_PLL_SYSCLK);
